README.md

```diff
@@ -11,7 +11,15 @@
 
 ## Overview
 
-AIConfig saves prompts, models and model parameters as source control friendly configs. This allows you to iterate on prompts and model parameters _separately from your application code_.
+AIConfig is an SDK that eases the development of LLM apps in two broad ways, each roughly corresponding to a class of use cases:
```

Comment:
This isn't the right spot for this nuance. The overview should convey the basic premise.

This nuance either belongs in an FAQ to start, or in a dedicated page of its own on the docs website.

Reply:
Fair enough, but I think the previous version of this is not doing justice to the library - we don't only separate code from config in a source controllable way; we also make it easier to do quick iteration in notebook/playground mode

Reply:
That's a good point. We can update the overview to show more of that value. It just has to be done in a succinct way